roles/openshift_cfme/files/openshift_cfme.exports

```diff
@@ -0,0 +1,3 @@
+/exports/miq-pv01 *(rw,no_root_squash,no_wdelay)
+/exports/miq-pv02 *(rw,no_root_squash,no_wdelay)
+/exports/miq-pv03 *(rw,no_root_squash,no_wdelay)
```

Comment:
These exports include the `no_root_squash` option because I **think** it's required based off of the [upstream docs](https://github.com/ManageIQ/manageiq-pods#make-persistent-volumes-to-host-the-miq-database-and-application-data) on the NFS configuration.

> Note: Recommended permissions for the PV volumes are 775, root uid/gid owned.

* If this can safely change to some other user, e.g.: `nfsnobody/nfsnobody` (`65534`/`65534`) then I'll happily change that.
* The `no_wdelay` param comes from [other upstream docs](https://docs.openshift.com/container-platform/3.5/install_config/persistent_storage/persistent_storage_nfs.html#nfs-export-settings) regarding optimal openshift NFS tuning.

>  The no_wdelay option prevents the server from delaying writes, which greatly improves read-after-write consistency.